pkg/validations/validations.go

```diff
@@ -76,6 +76,9 @@ func AllStrings(vs []string, f func(string) bool) bool {
 }
 
 func ValidateAdditionalNTPSource(commaSeparatedNTPSources string) bool {
+	if commaSeparatedNTPSources == "" {
+		return false
```

Comment:
Why isn't empty valid?  Isn't it necessary for unsetting the field?

Reply:
Yeah, but it's actually not relevant for this PR:) I'll remove it.